src/Symfony/Component/DependencyInjection/Tests/Loader/XmlFileLoaderTest.php

```diff
@@ -486,4 +486,14 @@ public function testAutowire()
 
         $this->assertTrue($container->getDefinition('bar')->isAutowired());
     }
+
+    /**
+     * @group legacy
+     */
+    public function testAliasDefinitionContainsUnsupportedElements()
+    {
+        $container = new ContainerBuilder();
+        $loader = new XmlFileLoader($container, new FileLocator(self::$fixturesPath.'/xml'));
+        $loader->load('legacy_invalid_alias_definition.xml');
```

Comment:
I'd also prefer to have an assertion. There's always an expected outcome. Not throwing an exception is not really the one. The fact that alias is registered in the container might be a better idea.

 Can we at least verify the alias was registered in the container please?